scripts/package/package_android_sdk.sh

```diff
@@ -23,9 +23,9 @@
 
 set -e
 
-ANDROID_PLATFORM=29
-ANDROID_TARGET_API_LEVEL=29
-ANDROID_BUILD_TOOLS_VERSION=29.0.3
+ANDROID_PLATFORM=30
+ANDROID_TARGET_API_LEVEL=30
+ANDROID_BUILD_TOOLS_VERSION=30.0.3
```

Comment:
DId we pick this exact ".0.3" at random, or did we just change the major number?